components/modal/subscriptions-modal/subscriptions-preview/table/component.js

```diff
@@ -17,6 +19,9 @@ class AlertsTable extends PureComponent {
     return (
       <div className="c-subscription-preview-table">
         <h3>{title}</h3>
+        <div className="disclaimer-container">
```

Comment:
Do you know if this kind of disclaimer is something we're going to spread in the app? Otherwise, we could isolate it in a component. If not, leave it as it is.

Reply:
Hmm That's a good question, I don't think we are right now, but let's take it into account and, in the case where we need to put a similar one again, we can create a component for this